cmd/account/get/account.go

```diff
@@ -117,10 +117,10 @@ func (o *getAccountOptions) run() error {
 			return err
 		}
 
-		for _, a := range accounts.Items {
+		for i, a := range accounts.Items {
 			if a.Spec.AwsAccountID == o.accountID {
 				accountCRName = a.Name
-				account = &a
+				account = &accounts.Items[i]
```

Comment:
I guess the idea here is to have `account` point an item of the list and not a copy of an item. This is great for consistency even if this was not causing any issue (except for `gosec`).
However each item of the list is still iteratively copied anyway in `a`; also code is not very consistent, what about:
```
for i := range accounts.Items {
    a := &accounts.Items[i]
    if a.Spec.AwsAccountID == o.accountID {
        accountCRName = a.Name
        account = &a
```

Reply:
Sorry didn't see your reply till now.
Yes I agree this change was mostly cosmetic, so not a must have indeed and anyway the PR has been merged without it.
I did not test if the code I proposed was compiling but if it was `*a.Name` and `a.Name` would point the same string so I do not see the risk.